pkg/crc/machine/kubeconfig.go

```diff
@@ -145,11 +147,16 @@ func addContext(cfg *api.Config, ip string, clusterConfig *types.ClusterConfig,
 				MinVersion: tls.VersionTLS12,
 			},
 			DialContext: func(ctx gocontext.Context, network, address string) (net.Conn, error) {
+				logging.Debugf("Using address: %s", address)
 				port := strings.SplitN(address, ":", 2)[1]
+				if strings.Contains(address, "oauth-openshift") {
```

Comment:
>  This is preexisting, but not clear why we always dial to ip when we have an explicit address. Just thinking out loud.

We are using IP just to avoid dns resolution in case it is not working or in proxied env where it is not excluded.